src/gwt/src/org/rstudio/studio/client/workbench/views/environment/dataimport/DataImportDialog.java

```diff
@@ -0,0 +1,28 @@
+package org.rstudio.studio.client.workbench.views.environment.dataimport;
+
+import org.rstudio.core.client.widget.ModalDialog;
+import org.rstudio.core.client.widget.OperationWithInput;
+
+import com.google.gwt.user.client.ui.Widget;
+
+public class DataImportDialog extends ModalDialog<String>
+{
+
+   public DataImportDialog(String caption, OperationWithInput<String> operation)
+   {
+      super(caption, operation);
+      super.setOkButtonCaption("Import");
```

Comment:
Perfect, will push PRs against v0.99-1000 and remove explicit qualification. Thanks JJ and Jonathan!